crypto/modes/asm/aes-gcm-avx512.pl

```diff
@@ -417,7 +411,7 @@ sub PROLOG {
 sub EPILOG {
   my ($hkeys_storage_on_stack, $payload_len) = @_;
 
-  my $rndsuffix = &random_string();
+  my $rndsuffix = "aesgcmavx512_" . $count++;
```

Comment:
Considering that it's just a suffix on local labels, this should be sufficient:

```suggestion
  my $rndsuffix = $count++;
```

Reply:
That is, of course, valid for all the spots where you made this change, not just here.

Reply:
Can I go further, and ask that this no longer be called `$rndsuffix` but either just `$suffix` or `$label_suffix`?

Let's try to have the code make as much sense in its new form as possible